### ISSUE TITLE
add runtime benchmarking to rules

### ISSUE THREAD

**Issue by kzuberi** [GeneMANIA/pipeline#22]
Looks like we need to add a command to the various rules to tell the build system to store benchmark stats in some file: [benchmark rules](https://bitbucket.org/johanneskoester/snakemake/wiki/Documentation#markdown-header-benchmark-rules)

So we could do something like:

~~~
rule RULE_A:
    input: whatever
    output: whatever
    benchmark: WORK+"/benchmarks/somecommand/RULE_A.json"
    shell: whatever
~~~

so basically adding that extra 'benchmark:" line to the rule. Unfortunately it looks like we have to do this for all rules we want to measure. I'd recommend trying it on a couple and having a look at the output file and deciding if this will be useful.

@haroldr you might be interested in trying this later.
